README.md

```diff
@@ -89,7 +89,7 @@ get all the requirements installed in one go.
   - Edge:
     [udemy_enroller_edge.py](https://github.com/aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE/blob/master/udemy_enroller_edge.py)
 
-- **Has issues:**
+- **Has issues when run on custom kernal but works fine on vanilla OS:**
```

Comment:
kernel ** :P

Reply:
I made the same typo in the comment for the PR as well :1st_place_medal:

Reply:
FIxed now :clap: